app/src/main/java/org/p2p/wallet/history/ui/token/adapter/holders/BridgePendingViewHolder.kt

```diff
@@ -19,14 +19,15 @@ class BridgePendingViewHolder(
     fun onBind(item: HistoryItem.BridgeSendItem) {
         itemView.setOnClickListener { onHistoryClicked(item) }
         with(binding) {
-            // TODO fix icons
-            transactionTokenImageView.apply {
-                ERC20Tokens.values()
-                    .firstOrNull { it.mintAddress == item.sendDetails.recipient.raw }
-                    ?.let { setTokenImage(glideManager, it.tokenIconUrl) }
-                    ?: setTransactionIcon(R.drawable.ic_transaction_send)
+            val foundToken = ERC20Tokens.values()
+                .firstOrNull { it.replaceTokenSymbol == item.sendDetails.amount.symbol }
```

Comment:
Again, we need a method for searching token with different params

Reply:
Yeah, we can do it. But for now I think it's useless

Reply:
Now or never

also values is a performance eating operation
https://stackoverflow.com/questions/2446135/is-there-a-performance-hit-when-using-enum-values-vs-string-arrays